### PR TITLE
Quebec income tax rate

### LINKED ISSUE #218: Quebec Income tax rate
Reference 
https://www.revenuquebec.ca/en/citizens/income-tax-return/completing-your-income-tax-return/income-tax-rates/

### PR DESCRIPTION
**Pull request by Shinuing**
Fixes #218

### CODE REVIEW

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/rate.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+description: Income tax rate in Quebec. 
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-CAD
+  rate_unit: /1
+  name: qc_income_tax_rate
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/rate.yaml:6]: Don't need name

File: policyengine_canada/tests/gov/provinces/qc/tax/income/income_tax.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+- name: Quebec income tax for 2023
```
Comment (PavelMakarchuk) [inline policyengine_canada/tests/gov/provinces/qc/tax/income/income_tax.yaml:1]: Individual names for each test please

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/rate.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+description: Income tax rate in Quebec.
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/rate.yaml:1]:
```suggestion
description: Quebec taxes income at these rates, depending on taxable income.
```

File: policyengine_canada/parameters/gov/provinces/qc/tax/income/rate.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+description: Quebec taxes income at these rates, depending on taxable income.
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-CAD
+  rate_unit: /1
+  label: Quebec income tax rates 
+  reference:
+    - title: Revenu Quebec - Income tax rate
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/qc/tax/income/rate.yaml:8]: Can you find a legislative reference?

Comment (Shinuing) [reply policyengine_canada/parameters/gov/provinces/qc/tax/income/rate.yaml:8]: The specific income tax rates for individuals are not set out in the text of the legislative Taxation Act (Quebec) , but typically published in the "Income Tax Rate" and on the website of Revenue Quebec.

**Pull request merged**